gammapy/irf/irf_stack.py

```diff
@@ -0,0 +1,94 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+import logging
+import numpy as np
+from astropy.units import Quantity
+from ..irf import EffectiveAreaTable, EnergyDispersion
+
+__all__ = [
+    'IRFstack',
+]
+
+log = logging.getLogger(__name__)
+
+
+class IRFstack(object):
```

Comment:
class names in gammapy are usually written in [CamelCase](https://en.wikipedia.org/wiki/Camel_case). So in this case I would propose ``IRFStacker`` analogous to ``SpectrumObservationStacker``.